### ISSUE TITLE
Feature request: Ability to select zone for loading/unloading when using multiple zones

### ISSUE THREAD

**Issue by joscha999** [loki79uk/FS25_UniversalAutoload#202]
While testing the fix for #196 I noticed sometimes logs are just the wrong thickness and I end up with differently packed loading zones or I just want to only load onto the front zone etc.

So when having multiple zones (like on that log trailer using two 6.2m log zones) it'd be nice to select a specific one to load to as well as selecting being able to select a specific one to unload from (and the rest would not unload).

I'm not 100% sure about potential conflicts but I'd throw in Ctrl + R/I as potential keybinds since Shift + R/I are already used to load.

**Comment by loki79uk** [comment]
Yes, should be easy enough, as you point out the hardest part is finding some spare key bindings for any new actions.  I won't get around to looking at this for a while, but will leave it open to remind me.
